### PR TITLE
Blue: Add placementId bidder param

### PR DESCRIPTION
**Pull request by samuelfabel**
This pull request refers to add a new parameter `placementId` at the bidder-params of Blue bidder.

## 🏷 Type of documentation
- [ ] new bid adapter
- [x] update bid adapter
- [ ] new feature
- [ ] text edit only (wording, typos)
- [ ] bugfix (code examples)
- [ ] new examples

## 📋 Checklist
- [x] Related pull requests in prebid.js or server are linked https://github.com/prebid/prebid.github.io/pull/5724
- [ ] For new adapters check [submitting your adapter docs](https://docs.prebid.org/dev-docs/bidder-adaptor.html#submitting-your-adapter)

### CODE REVIEW

File: static/bidder-params/blue.json
Comment (bsardo) [inline static/bidder-params/blue.json:1]: Please open up a docs PR and leave the link on this PR.

Comment (bsardo) [inline static/bidder-params/blue.json:1]: I don't see a `adapters/blue/params_test.go` file. Please add it to ensure your bidder params rules defined in your `static/bidder-params/blue.json` file are working properly. I suggest copying it from another adapter as a starting point.

Comment (samuelfabel) [reply static/bidder-params/blue.json:1]: done

Comment (samuelfabel) [reply static/bidder-params/blue.json:1]: done

File: adapters/blue/params_test.go [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+}
+
+var invalidParams = []string{
+	``,
+	`null`,
+	`true`,
+	`5`,
+	`4.2`,
+	`[]`,
+	`{}`,
```
Comment (bsardo) [inline adapters/blue/params_test.go:49]: Please add a test case where `placementId` exists but `publisherId` does not.

File: adapters/blue/params_test.go [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+	for _, invalidParam := range invalidParams {
+		if err := validator.Validate(openrtb_ext.BidderBlue, json.RawMessage(invalidParam)); err == nil {
+			t.Errorf("Schema allowed unexpected params: %s", invalidParam)
+		}
+	}
+}
+
+var validParams = []string{
+	`{"publisherId":"1234"}`,
+	`{"publisherId":"1234", "placementId":"12345"}`,
```
Comment (bsardo) [inline adapters/blue/params_test.go:39]: From looking at your JSON bidder params file, you don't have any minimum length requirements on these fields. In that case, please add a test case(s) where these fields are empty strings.

File: adapters/blue/params_test.go [hunk 3]
```diff
@@ -0,0 +1,50 @@
+package blue
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/prebid/prebid-server/v3/openrtb_ext"
+)
+
+// This file actually intends to test static/bidder-params/blue.json TODO: MUST BE CREATED
```
Comment (bsardo) [inline adapters/blue/params_test.go:10]: Please delete the TODO part of this comment or the entire comment.

File: adapters/blue/params_test.go (hunk 1 above)
Comment (samuelfabel) [reply adapters/blue/params_test.go:49]: done

File: adapters/blue/params_test.go (hunk 2 above)
Comment (samuelfabel) [reply adapters/blue/params_test.go:38]: done

File: adapters/blue/params_test.go (hunk 3 above)
Comment (samuelfabel) [reply adapters/blue/params_test.go]: done

**Review by bsardo — APPROVED** [review]

**Review by przemkaczmarek — APPROVED** [review]

**Pull request merged**